package.yaml

```diff
@@ -10,8 +10,7 @@ copyright: "2021 Renaissance Learning Inc"
 extra-source-files:
   - README.md
   - CHANGELOG.md
-  - tests/golden.xml
-  - tests/golden-prefixed.xml
+  - tests/*.xml
```

Comment:
I'm not sure it matters, but `golden*.xml` felt safer than any xml files whatsoever. For example, perhaps we generate our own suite's xml reports into `tests/` some day, we wouldn't want them being caught here and generating cabal file noise.

I'll Approve anyway, but (if you agree) feel free to change this if you get to it before the test suite runs and auto-merge happens. If not, no worries.